scripts/get_bank_registry_it.py

```diff
@@ -0,0 +1,74 @@
+import json
+from time import sleep
+
+import requests
+from bs4 import BeautifulSoup
+
+
+def get_banks_registry_data_from_bank_name(bank_name):
+    sleep(1)  # prevent server DoSing
+
+    url = 'https://www.ibancalculator.com/blz.html'
+    data = {
+        'tx_blz_pi1[country]': 'IT',
+        'tx_blz_pi1[searchterms]': bank_name,
+        'tx_blz_pi1[bankcode]': '',
+        'tx_blz_pi1[fi]': 'fi',
+        'no_cache': 1,
+        'Action': 'Search'
+    }
+
+    response = requests.post(url, data=data)
+    soup = BeautifulSoup(response.content, "html.parser")
+
+    results_tables = soup.select(".table")
+
+    if results_tables:
+        for row in results_tables[0].select("tr")[1:]:
+            bank_code = row.select("td")[3].text
+            bic = row.select("td")[2].text
+            if bank_code and bic and row.select("td")[0].text == "IT":
+                yield {
+                    'country_code': 'IT',
+                    'primary': True,
+                    'bic': str(bic).split(',')[0],
+                    'bank_code': str(int(bank_code)).zfill(5),
+                    'name': row.select("td")[1].text,
+                    'short_name': row.select("td")[1].text,
```

Comment:
I've created a function which parses the short_name from the bank name. I've also added tests for the function to the generator script in case there is some other case to be parsed. Could you please review it again?